### DISCUSSION TITLE
Allow public Discussions on private repos and bring Discussions to GitHub Desktop

### DISCUSSION THREAD

**Discussion by rafaeldjpbrochado** [community/community#4996] [Discussions]
Two suggestions:

- Consider allowing private repos with public Discussion forums. Could be a way for us developers to discuss upcoming specs/features with our community even if the code for the thing we're working on is closed-source.

- Please consider upgrading the GitHub desktop app to bring to it all the functionality we already have access to on the web - especially Discussions.

**Comment by tsal** [comment]
I'm definitely a fan of the second suggestion, but I'd go further and say support them in the mobile apps, too.

**Comment by douglasg14b** [comment]
Public discussions for a private repo is a great tool for community engagements while keeping everything in the same repo.
